src/generate/handlebarsAmfHelpersGetType.test.ts

```diff
@@ -0,0 +1,274 @@
+/*
```

Comment:
Why are these tests split out into its own file?

Reply:
Otherwise the file becomes too big, bigger than our current limit of 500 lines.